src/common/Utilities/Util.cpp

```diff
@@ -611,7 +611,7 @@ bool Utf8ToUpperOnlyLatin(std::string& utf8String)
     return WStrToUtf8(wstr, utf8String);
 }
 
-std::string ByteArrayToHexStr(uint8 const* bytes, uint32 arrayLen, bool reverse /* = false */)
+std::string ByteArrayToHexStr(uint8 const* bytes, size_t arrayLen, bool reverse /* = false */)
```

Comment:
cue anti-boost protests

`boost::algorithm::(un)hex`?

Reply:
sure, i honestly just wanted to replace the parameter, we can change that